node-require/add.js

```diff
@@ -0,0 +1,5 @@
+
+module.exports = function add(num1, num2) {
+  const sum = num1 + num2;
+  console.log('result: ', sum);
```

Comment:
Your functions should `return` their results instead of logging them. The logging should happen in `calculate.js`.